exercises/twelve-days/canonical-data.json

```diff
@@ -29,7 +29,7 @@
             "endVerse": 2
           },
           "expected": [
-            "On the second day of Christmas my true love gave to me, two Turtle Doves, and a Partridge in a Pear Tree."
+            "On the second day of Christmas my true love gave to me: two Turtle Doves, and a Partridge in a Pear Tree."
```

Comment:
`two`, `three`, `four` etc. should all also be capitalized if want to be really picky.
https://en.wikipedia.org/wiki/The_Twelve_Days_of_Christmas_(song)#/media/File:12_days_melody.png

Reply:
Personally I think it'd be better to lowercase all of the items.